conda/environments/streamz_dev.yml

```diff
@@ -0,0 +1,23 @@
+name: streamz_dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python>=3.7
+  - cython>=0.29,<0.30
```

Comment:
Where is the cython dep coming from?

Reply:
cytoolz? I don't suppose we're building anything.

Reply:
Right, if one of the deps needs it conda should figure that out.

Reply:
ahh ... my bad I meant to remove that. I was experimenting with implementing librdkafka with cython and forgot to remove it. I will remove them now.

On a side note that is working very well BTW and might open something up around that as well. Working on the best way to make cython not a hard dependency at the moment since I'm sure several people don't using streamz don't necessarily want it.

Reply:
> Working on the best way to make cython not a hard dependency

Typically you can include the C file for those that want to compile (source dist) and of course, if you build wheels and conda packages, those should not need cython.

Reply:
I'm confused, do you mean that you are using librdkafka with cython or using cython to speed up `streamz` kafka bindings?

Reply:
> I'm confused, do you mean that you are using librdkafka with cython or using cython to speed up `streamz` kafka bindings?

Actually both. I was replacing the current kafka sources with my own sources (not the confluent-python-kafka library) which were cython modules that interacted directly with librdkafka++. There are many many advantages to that especially in the realm of partition balancing that the new KafkaConsumer offers (not implemented in the confluent python library yet) for distributed systems with many processes appearing/disappearing.

In a nutshell I was seeing about 15-20X speedups in my consumers when using my cython implementations.